### PR TITLE
fixing premolar and molar teeth

### PR DESCRIPTION
**Pull request by megbalk**
I fixed definitions and made sure there were correct subclasses being inferred for all the premolars and molars. I did not do anything with the deciduous teeth. I also added abbreviations to permanent premolar and molars.

### CODE REVIEW

File: uberon_edit.obo [hunk 1]
```diff
@@ -193632,15 +193642,6 @@ intersection_of: UBERON:0006983 ! anatomical point
 intersection_of: distalmost_part_of UBERON:0002415 ! tail
 created_by: https://orcid.org/0000-0003-2699-3066
 
-[Term]
-id: UBERON:7500036
```
Comment (cmungall) [inline uberon_edit.obo:193636]: Hi, it looks like you deleted this term. Was this deliberate? It looks like you added this in the first place so I will leave it to you whether or not you want to keep it - however, we have a policy of no deletions, IDs must be retained. You can use the obsolete entity (GO workflow) option in Protege to obsolete this.

File: uberon_edit.obo [hunk 2]
```diff
@@ -193844,10 +193848,11 @@ created_by: https://orcid.org/0000-0003-2699-3066
 id: UBERON:7500059
 name: upper third secondary premolar tooth
 def: "The premolar tooth of the upper jaw that is phylogenetically number 3." [https://orcid.org/0000-0003-2699-3066]
-synonym: "P3" RELATED []
+synonym: "P3" RELATED ABBREVIATION []
 synonym: "upper third permanent premolar tooth" EXACT []
-is_a: UBERON:0016944 ! upper premolar tooth
-is_a: UBERON:0017270 ! secondary premolar tooth
+intersection_of: UBERON:7500056 ! premolar 3
```
Comment (cmungall) [inline uberon_edit.obo:193853]: everything else looks great!

**Comment by megbalk** [comment]
@cmungall yes, it was an accident to add it, and I deleted it before I read the thread about obsolete terms. I can go back and do that...I'm not sure how though. Could I get some help?

**Comment by nicolevasilevsky** [comment]
Hi @megbalk I think I can help you obsolete this term. Do you want to have a call and I can walk you through it? Can you email me at: vasilevs at ohsu dot edu?

**Comment by megbalk** [comment]
I think my changes should be there. I have made UBERON:7500036 obsolete. Does that show up?

**Comment by nicolevasilevsky** [comment]
@megbalk I don't see your changes, I suspect that you did not commit your changes to this branch. Do you know what I mean? I'm happy to jump on another call and we can troubleshoot.

File: uberon_edit.obo (hunk 1 above)
Comment (megbalk) [reply uberon_edit.obo:193636]: @cmungall did my changes come through?

**Comment by ramonawalls** [comment]
I *think* this branch has been fixed. @megbalk Can you resolve the conflicts, or can @nicolevasilevsky  show you how to so we can merge?

**Comment by nicolevasilevsky** [comment]
@megbalk I'm happy to work on this with you - let me know :)

**Comment by megbalk** [comment]
@nicolevasilevsky that'd be great!

**Pull request closed without merging**